cli/common.go

```diff
@@ -74,13 +80,14 @@ func createSliceIDs(slices []string) ([]string, error) {
 	return sliceIDs, nil
 }
 
-func createRequest(slices []string) (controller.Request, error) {
+func createRequestWithContent(slices []string) (controller.Request, error) {
 	err := validateArgs(slices)
 	if err != nil {
 		return controller.Request{}, maskAny(err)
 	}
 
 	req := controller.Request{
+		Group:    dirnameFromSlices(slices),
```

Comment:
Yeah this is no way around. Funny you mentioned the need of it just a few hours ago.